shared/chat/conversation/giphy/container.tsx

```diff
@@ -6,19 +7,21 @@ import GiphySearch from '.'
 
 type OwnProps = {conversationIDKey: Types.ConversationIDKey}
 
-export default Container.connect(
```

Comment:
a lot of older connects got modernized to useSelect so we can get access to hooks and to avoid thrash